include/maths/CBoostedTreeLeafNodeStatistics.h

```diff
@@ -236,7 +257,6 @@ class MATHS_EXPORT CBoostedTreeLeafNodeStatistics final {
         void swap(CSplitsDerivatives& other) {
             std::swap(m_NumberLossParameters, other.m_NumberLossParameters);
             m_Derivatives.swap(other.m_Derivatives);
-            m_MissingDerivatives.swap(other.m_MissingDerivatives);
```

Comment:
Don't you need to handle missing derivative extra anymore?

Reply:
I've just put them at the end of `m_Derivatives`. This was convenient because I need to assign an integer for "this is missing" so I just use # splits + 1 (conveniently it takes all branching out of the hot method `addRowDerivatives`)